packages/i18n/package.json

```
@@ -0,0 +1,33 @@
+{
+  "name": "@node-core/website-i18n",
+  "type": "module",
+  "exports": {
+    ".": {
+      "types": "./src/index.d.ts",
+      "default": "./dist/index.js"
+    },
+    "./types": {
+      "types": "./src/types.d.ts",
+      "default": "./dist/types.js"
+    },
+    "./config.json": "./dist/config.json",
+    "./locales/en.json": "./locales/en.json"
+  },
+  "scripts": {
+    "build": "tsc",
+    "dev": "tsc --watch",
+    "lint:js": "eslint src/**/*.ts --cache --cache-strategy=content --cache-location=.eslintjscache"
+  },
+  "dependencies": {
+    "@typescript-eslint/eslint-plugin": "7.11.0",
+    "@typescript-eslint/parser": "7.11.0",
```

This is an older major version. We released v8 a few weeks back (https://typescript-eslint.io/blog/announcing-typescript-eslint-v8): 

```suggestion
    "@typescript-eslint/eslint-plugin": "8.4.0",
    "@typescript-eslint/parser": "8.4.0",
```

This is also why `npm run lint` produces complaints about TypeScript versions:

```plaintext

@nodejs/website:lint:js: 
@nodejs/website:lint:js: WARNING: You are currently running a version of TypeScript which is not officially supported by @typescript-eslint/typescript-estree.
@nodejs/website:lint:js: 
@nodejs/website:lint:js: You may find that it works just fine, or you may not.
@nodejs/website:lint:js: 
@nodejs/website:lint:js: SUPPORTED TYPESCRIPT VERSIONS: >=4.7.4 <5.5.0
@nodejs/website:lint:js: 
@nodejs/website:lint:js: YOUR TYPESCRIPT VERSION: 5.5.4
@nodejs/website:lint:js: 
@nodejs/website:lint:js: Please only submit bug reports when using the officially supported version.
@nodejs/website:lint:js: 
@nodejs/website:lint:js: =============
```

I see the rest of the repo is on older versions, so I'll send a PR. 🙂